qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/DiagnosticDataFactory.java

```diff
@@ -43,6 +43,13 @@ public static JsonObject createUndefinedSectionTagData(String tagName) {
 		return data;
 	}
 
+	public static JsonObject createMissingMemberData(String signature, String property) {
+		JsonObject data = new JsonObject();
+		data.addProperty("signature", signature);

```

Comment:
Try to create a class like UnkwownProperryData with signature and property fields instead of using JsonObject

Reply:
sorry, I did that, I just forgot to squash the commit

Reply:
I think UnkwownProperryData is more generic for nzme znd @AlexXuChen could use it for the similar codeaction